base/shell/explorer/explorer.cpp

```diff
@@ -154,6 +154,8 @@ StartWithDesktop(IN HINSTANCE hInstance)
     buttons to work right) */
     HideMinimizedWindows(TRUE);
 
+    ProcessRunOnceItems(); // Must be executed before the desktop is created
```

Comment:
Does this need to be done after the tray window creation or could it be done before?

Reply:
I'd rather it be done before but I did not want to move the HideMinimizedWindows call and deal with the #ifdef. For the issue with internet icon, the current location is enough. I did not try WinSpy on Windows to see if the tray is there but there is certainly nothing visible during RunOnce.

Reply:
Good enough for me :+1: